src/smc-webapp/jupyter/actions.coffee

```diff
@@ -75,6 +75,11 @@ class exports.JupyterActions extends Actions
         store._is_project = @_is_project
         @_account_id = client.client_id()   # project or account's id
 
+        # this initializes actions+store for the assistant -- are "sub-actions" a thing?
+        if not @_is_project   # this is also only a UI specific action
+            {instantiate_assistant} = require('../assistant/main')
+            @assistant_actions = instantiate_assistant(project_id, path)
```

Comment:
Hmmm, they aren't a thing in our code base. But I'm not sure I see anything wrong with it. Especially since several editor actions would want their own examples store/actions right? The alternative would be a global actions class that handles several stores but that seems like it could get unwieldy fast.